### PR TITLE
renamed openverse-api directory to openverse_api

### LINKED ISSUE #223: [Bug] Directory naming consistiency
We should rename the root `openverse-api` directory to `openverse_api` to match the naming convention of other root-level directories.

### PR DESCRIPTION
**Pull request by MuhammadFaizanHaidar**
## Fixes

Fixes #223 by @zackkrida

## Description
Added Directory naming consistency.

## Technical details
renamed openverse-api directory to openverse_api.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by zackkrida — APPROVED** [review]

**Comment by obulat** [comment]
The tests are failing, probably because you also need to change this line:
https://github.com/WordPress/openverse-api/blob/a470f3f3241c6c16cc208a8c1c579726ab12f5b3/docker-compose.yml#L55

**Review by obulat — CHANGES_REQUESTED** [review]
Please, try to change the line I mentioned in the dockerfile, and see if the tests in checks pass.

**Comment by obulat** [comment]
Thank you for quick changes :) 
Now, there is a problem with the GitHub actions check because it's still looking for files in `openverse-api`:
https://github.com/WordPress/openverse-api/blob/a470f3f3241c6c16cc208a8c1c579726ab12f5b3/.github/workflows/integration-tests.yml#L21
There are several places in that file where you need to change `openverse-api` to `openverse_api`

**Comment by MuhammadFaizanHaidar** [comment]
> Thank you for quick changes :)
> Now, there is a problem with the GitHub actions check because it's still looking for files in `openverse-api`:
> https://github.com/WordPress/openverse-api/blob/a470f3f3241c6c16cc208a8c1c579726ab12f5b3/.github/workflows/integration-tests.yml#L21
> 
> 
> There are several places in that file where you need to change `openverse-api` to `openverse_api`

sure 👍🏻

**Comment by MuhammadFaizanHaidar** [comment]
> > Thank you for quick changes :)
> > Now, there is a problem with the GitHub actions check because it's still looking for files in `openverse-api`:
> > https://github.com/WordPress/openverse-api/blob/a470f3f3241c6c16cc208a8c1c579726ab12f5b3/.github/workflows/integration-tests.yml#L21
> > 
> > There are several places in that file where you need to change `openverse-api` to `openverse_api`
> 
> sure 👍🏻

Do I need to update all openverse-api occurrence in this file? on just mentioned line number?

**Comment by MuhammadFaizanHaidar** [comment]
@obulat its still looking with wrong directory name.

**Comment by zackkrida** [comment]
Hello @MuhammadFaizanHaidar, I was able to create a list of all the places where `openverse-api` is still referenced. All of these need to be renamed to `openverse_api`. This is such a great example of how a seemingly simple change can have many effects and be more complicated than you might think!

```bash
api on  openverse_api_dr_renamed +1/-1 [$!?]
❯ rg 'openverse-api'
CONTRIBUTORS.md
1:# openverse-api contributors (sorted alphabetically by last name)

CONTRIBUTING.md
15:If you find a bug, please open an issue in this repository describing the bug. You can file a bug [here](https://github.com/wordpress/openverse-api/issues/new?template=bug_report.md). You will see a bug report template with the required information you should provide.
21:If you have an idea of a new feature or change to how the Openverse API works, please [start a discussion](https://github.com/WordPress/openverse-api/discussions/new?category=ideas) so we can discuss the possibility of that change or new feature being implemented and released in the future. This lets us come to an agreement about the proposed idea before any work is done.
27:If you want to get started contributing code to this project but don't know exactly what to work on, we compiled a good list of issues labeled as [`good first issue`](https://github.com/wordpress/openverse-api/labels/good%20first%20issue) which are small in scope and not so complex to solve. There are also issues labeled as [`help wanted`](https://github.com/wordpress/openverse-api/labels/help%20wanted) which can be a bit more complex but are good examples of things we are currently accepting help from the community.
91:Feel free to [join us on Slack](https://make.wordpress.org/chat/) and discuss the project with the engineers and community memebers on #openverse. We also use [GitHub discussions](https://github.com/WordPress/openverse-api/discussions) for feature requests, troubleshooting, and other 'off-topic' conversations.

analytics/docs/swagger.yaml
11:    url: "https://github.com/wordpress/openverse-api/blob/master/LICENSE"

openverse_api/catalog/settings.py
72:        'LOGOS_BUCKET', 'openverse-api-logos-prod'

openverse_api/catalog/urls/swagger.py
18:              "WordPress/openverse-api/blob/master/LICENSE"

openverse_api/catalog/api/docs/README.md
3:The Openverse API ('openverse-api') is a system that allows programmatic access
101:[contributing on Github](https://github.com/wordpress/openverse-api), here’s a
111:  [link](https://github.com/wordpress/openverse-api#running-the-server-locally)
114:  [file](https://github.com/wordpress/openverse-api/blob/master/.github/workflows/integration-tests.yml)

README.md
5:The Openverse API (`openverse-api`) is a system that allows programmatic access to public domain digital media. It is our ambition to index and catalog [billions of openly-licensed works](https://stateof.creativecommons.org/), including articles, songs, videos, photographs, paintings, and more. Using this API, developers will be able to access the digital commons in their own applications.
26:git clone https://github.com/WordPress/openverse-api.git
97:- _openverse-api_ is a Django Rest Framework API server. For a full description of its capabilities, please see the [browsable documentation](https://api.openverse.engineering).
173:You are welcome to take any open issue in the tracker labeled [`help wanted`](https://github.com/WordPress/openverse-api/labels/help%20wanted) or [`good first issue`](https://github.com/WordPress/openverse-api/labels/good%20first%20issue); **there's no need to ask for permission in advance**. Other issues are open for contribution as well, but may be less accessible or well-defined in comparison to those that are explicitly labeled.

DOCUMENTATION_GUIDELINES.md
15:- Run the server locally by following this [link](https://github.com/wordpress/openverse-api#running-the-server-locally)
17:- Make sure the updates passed the automated tests in this [file](https://github.com/wordpress/openverse-api/blob/master/.github/workflows/integration-tests.yml)

openverse_api/catalog/api/utils/scheduled_tasks.py
14:inside of the cache (shared by all instances of openverse-api).

ingestion_server/README.md
17:![How indexing works](https://github.com/wordpress/openverse-api/blob/master/ingestion_server/howitworks.png)

ingestion_server/ingestion_server/categorize.py
4:https://github.com/wordpress/openverse-api/issues/340
```

**Comment by dhruvkb** [comment]
Also, there are a large number of merge conflicts after #224.

**Comment by MuhammadFaizanHaidar** [comment]
> Hello @MuhammadFaizanHaidar, I was able to create a list of all the places where `openverse-api` is still referenced. All of these need to be renamed to `openverse_api`. This is such a great example of how a seemingly simple change can have many effects and be more complicated than you might think!
> 
> ```shell
> api on  openverse_api_dr_renamed +1/-1 [$!?]
> ❯ rg 'openverse-api'
> CONTRIBUTORS.md
> 1:# openverse-api contributors (sorted alphabetically by last name)
> 
> CONTRIBUTING.md
> 15:If you find a bug, please open an issue in this repository describing the bug. You can file a bug [here](https://github.com/wordpress/openverse-api/issues/new?template=bug_report.md). You will see a bug report template with the required information you should provide.
> 21:If you have an idea of a new feature or change to how the Openverse API works, please [start a discussion](https://github.com/WordPress/openverse-api/discussions/new?category=ideas) so we can discuss the possibility of that change or new feature being implemented and released in the future. This lets us come to an agreement about the proposed idea before any work is done.
> 27:If you want to get started contributing code to this project but don't know exactly what to work on, we compiled a good list of issues labeled as [`good first issue`](https://github.com/wordpress/openverse-api/labels/good%20first%20issue) which are small in scope and not so complex to solve. There are also issues labeled as [`help wanted`](https://github.com/wordpress/openverse-api/labels/help%20wanted) which can be a bit more complex but are good examples of things we are currently accepting help from the community.
> 91:Feel free to [join us on Slack](https://make.wordpress.org/chat/) and discuss the project with the engineers and community memebers on #openverse. We also use [GitHub discussions](https://github.com/WordPress/openverse-api/discussions) for feature requests, troubleshooting, and other 'off-topic' conversations.
> 
> analytics/docs/swagger.yaml
> 11:    url: "https://github.com/wordpress/openverse-api/blob/master/LICENSE"
> 
> openverse_api/catalog/settings.py
> 72:        'LOGOS_BUCKET', 'openverse-api-logos-prod'
> 
> openverse_api/catalog/urls/swagger.py
> 18:              "WordPress/openverse-api/blob/master/LICENSE"
> 
> openverse_api/catalog/api/docs/README.md
> 3:The Openverse API ('openverse-api') is a system that allows programmatic access
> 101:[contributing on Github](https://github.com/wordpress/openverse-api), here’s a
> 111:  [link](https://github.com/wordpress/openverse-api#running-the-server-locally)
> 114:  [file](https://github.com/wordpress/openverse-api/blob/master/.github/workflows/integration-tests.yml)
> 
> README.md
> 5:The Openverse API (`openverse-api`) is a system that allows programmatic access to public domain digital media. It is our ambition to index and catalog [billions of openly-licensed works](https://stateof.creativecommons.org/), including articles, songs, videos, photographs, paintings, and more. Using this API, developers will be able to access the digital commons in their own applications.
> 26:git clone https://github.com/WordPress/openverse-api.git
> 97:- _openverse-api_ is a Django Rest Framework API server. For a full description of its capabilities, please see the [browsable documentation](https://api.openverse.engineering).
> 173:You are welcome to take any open issue in the tracker labeled [`help wanted`](https://github.com/WordPress/openverse-api/labels/help%20wanted) or [`good first issue`](https://github.com/WordPress/openverse-api/labels/good%20first%20issue); **there's no need to ask for permission in advance**. Other issues are open for contribution as well, but may be less accessible or well-defined in comparison to those that are explicitly labeled.
> 
> DOCUMENTATION_GUIDELINES.md
> 15:- Run the server locally by following this [link](https://github.com/wordpress/openverse-api#running-the-server-locally)
> 17:- Make sure the updates passed the automated tests in this [file](https://github.com/wordpress/openverse-api/blob/master/.github/workflows/integration-tests.yml)
> 
> openverse_api/catalog/api/utils/scheduled_tasks.py
> 14:inside of the cache (shared by all instances of openverse-api).
> 
> ingestion_server/README.md
> 17:How indexing works
> 
> ingestion_server/ingestion_server/categorize.py
> 4:https://github.com/wordpress/openverse-api/issues/340
> ```

@zackkrida I have replaced all those occurrences with openverse_api now.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,19 +12,19 @@ Please consult the [README](./README.md) and [CODEBASE](./CODEBASE.md) files at
...
+If you find a bug, please open an issue in this repository describing the bug. You can file a bug [here](https://github.com/wordpress/openverse_api/issues/new?template=bug_report.md). You will see a bug report template with the required information you should provide.
 
 After that, don't forget to tag the issue with the "Bug" label.
 
 ### Proposing changes or new features
 
-If you have an idea of a new feature or change to how the Openverse API works, please [start a discussion](https://github.com/WordPress/openverse-api/discussions/new?category=ideas) so we can discuss the possibility of that change or new feature being implemented and released in the future. This lets us come to an agreement about the proposed idea before any work is done.
+If you have an idea of a new feature or change to how the Openverse API works, please [start a discussion](https://github.com/WordPress/openverse_api/discussions/new?category=ideas) so we can discuss the possibility of that change or new feature being implemented and released in the future. This lets us come to an agreement about the proposed idea before any work is done.
```
Comment (obulat) [inline CONTRIBUTING.md:21]: This is the name of the Github repository, and it should still stay the same, `openverse-api`

File: DOCUMENTATION_GUIDELINES.md [hunk 2]
```diff
@@ -12,9 +12,9 @@ Openverse API uses [drf-yasg](https://github.com/axnsan12/drf-yasg), which is a
 
 ## How to Start Contributing
 
-- Run the server locally by following this [link](https://github.com/wordpress/openverse-api#running-the-server-locally)
+- Run the server locally by following this [link](https://github.com/wordpress/openverse_api#running-the-server-locally)
```
Comment (obulat) [inline DOCUMENTATION_GUIDELINES.md:15]:
```suggestion
- Run the server locally by following this [link](https://github.com/WordPress/openverse-api#running-the-server-locally)
```

File: analytics/docs/swagger.yaml [hunk 3]
```diff
@@ -8,7 +8,7 @@ info:
     email: "zack.krida@automattic.com"
   license:
     name: "MIT License"
-    url: "https://github.com/wordpress/openverse-api/blob/master/LICENSE"
+    url: "https://github.com/wordpress/openverse_api/blob/master/LICENSE"
```
Comment (obulat) [inline analytics/docs/swagger.yaml:11]:
```suggestion
    url: "https://github.com/wordpress/openverse-api/blob/master/LICENSE"
```

File: ingestion_server/README.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ Performance is dependent on the size of the target Elasticsearch cluster, databa
 
 ## How Indexing Works
 
-![How indexing works](https://github.com/wordpress/openverse-api/blob/master/ingestion_server/howitworks.png)
+![How indexing works](https://github.com/wordpress/openverse_api/blob/master/ingestion_server/howitworks.png)
```
Comment (obulat) [inline ingestion_server/README.md:17]:
```suggestion
![How indexing works](https://github.com/wordpress/openverse-api/blob/master/ingestion_server/howitworks.png)
```

File: .github/release-drafter.yml [hunk 5]
```diff
@@ -32,4 +32,4 @@ template: |
 
   ## See an issue?
 
-  Does anything look wrong in this release? Please [report a bug](https://github.com/WordPress/openverse-api/issues/new?labels=bug%2C+awaiting+triage&template=bug_report.md).
+  Does anything look wrong in this release? Please [report a bug](https://github.com/WordPress/openverse_api/issues/new?labels=bug%2C+awaiting+triage&template=bug_report.md).
```
Comment (obulat) [inline .github/release-drafter.yml:35]:
```suggestion
  Does anything look wrong in this release? Please [report a bug](https://github.com/WordPress/openverse-api/issues/new?labels=bug%2C+awaiting+triage&template=bug_report.md).
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -12,19 +12,19 @@ Please consult the [README](./README.md) and [CODEBASE](./CODEBASE.md) files at
 
 ### Bugs
 
-If you find a bug, please open an issue in this repository describing the bug. You can file a bug [here](https://github.com/wordpress/openverse-api/issues/new?template=bug_report.md). You will see a bug report template with the required information you should provide.
+If you find a bug, please open an issue in this repository describing the bug. You can file a bug [here](https://github.com/wordpress/openverse_api/issues/new?template=bug_report.md). You will see a bug report template with the required information you should provide.
```
Comment (obulat) [inline CONTRIBUTING.md:15]:
```suggestion
If you find a bug, please open an issue in this repository describing the bug. You can file a bug [here](https://github.com/WordPress/openverse-api/issues/new?template=bug_report.md). You will see a bug report template with the required information you should provide.
```

File: openverse_api/catalog/api/docs/README.md [hunk 7]
```diff
@@ -108,9 +108,9 @@ todo list to get started.
 - Read up about Documentation Guidelines, which provides guidelines on how to
   contribute to documentation, documentation styles and cheat sheet for drf-yasg
 - Run the server locally by following this
-  [link](https://github.com/wordpress/openverse-api#running-the-server-locally)
+  [link](https://github.com/wordpress/openverse_api#running-the-server-locally)
 - Update documentation or codebase
 - Make sure the updates passed the automated tests in this
-  [file](https://github.com/wordpress/openverse-api/blob/master/.github/workflows/integration-tests.yml)
+  [file](https://github.com/wordpress/openverse_api/blob/master/.github/workflows/integration-tests.yml)
```
Comment (obulat) [inline openverse_api/catalog/api/docs/README.md:114]:
```suggestion
  [file](https://github.com/WordPress/openverse-api/blob/master/.github/workflows/integration-tests.yml)
```

**Review by obulat — CHANGES_REQUESTED** [review]
Thank you so much for such detailed changes! This PR definitely turned out larger than I expected :)
The repository name on GitHub still remains `WordPress/openverse-api`, with a dash and not underscore, so I've added code suggestions inline to change the GitHub URLs back. You can simply commit them in batch, and then this PR should be ready.

File: openverse_api/catalog/api/docs/README.md [hunk 8]
```diff
@@ -98,7 +98,7 @@ $ curl \
 # Contribute
 
 We love pull requests! If you’re interested in
-[contributing on Github](https://github.com/wordpress/openverse-api), here’s a
+[contributing on Github](https://github.com/wordpress/openverse_api), here’s a
```
Comment (obulat) [inline openverse_api/catalog/api/docs/README.md:101]:
```suggestion
[contributing on Github](https://github.com/wordpress/openverse-api), here’s a
```

**Comment by dhruvkb** [comment]
@MuhammadFaizanHaidar this was fixed in 608b63c.

File: CONTRIBUTORS.md [hunk 9]
```diff
@@ -1,4 +1,4 @@
-# openverse-api contributors (sorted alphabetically by last name)
+# openverse_api contributors (sorted alphabetically by last name)
```
Comment (dhruvkb) [inline CONTRIBUTORS.md:1]: This is the repo name

**Review by dhruvkb — DISMISSED** [review]

**Review by obulat — APPROVED** [review]

**Comment by obulat** [comment]
Thank you for your contribution and your persistence, @MuhammadFaizanHaidar !

**Comment by MuhammadFaizanHaidar** [comment]
> Thank you for your contribution and your persistence, @MuhammadFaizanHaidar !

@obulat pleasure is all mine.

**Pull request merged**